docs/configure/triggers.md

```diff
@@ -46,22 +46,28 @@ For example, if you want to trigger a trigger on temperature readings, configure
 {{% /alert %}}
 
 {{% /tab %}}
```

Comment:
So I think the create menu option doesn't need the tabs here because it's now just one item. I am wondering if it wouldn't be better to get rid of the `Builder mode: Create menu` tab entirely since it's a shortcut for the other one and hopefully the rest should explain itself? I think the copy for adding webhooks & emails is better here though so merging that with the other tab would be good

Reply:
I was thinking about the other webhook option that's about to be documented, then there'll be a second tab. And i'm fine removing that, will do now

Reply:
but also wait are you sure you want to remove that? in my understanding those are their own types of triggers and you can't configure them otherwise/with plain builder mode, its not super self explanatory that you can do that. maybe if the conditional data ingested is the same thing but I'm not sure

Reply:
You can show me tomorrow in the UI and we can see if you can create the same trigger with just the add trigger ui

Reply:
removed!